benchmark/benchmark_utils.py

```diff
@@ -96,6 +101,47 @@ def return_or_create_result_file(
     return results
 
 
+def return_or_create_wrong_result_file(task: str):
```

Comment:
This is not an intuitive name for the function. I had trouble understanding what it is for just from reading the code (what is a "wrong result file"). I would suggest naming the entire process something like "failure mode identification" such that it is intuitively clear what is happening. What you are doing is saving responses in case of a failure.

Reply:
I changed the naming in [c137e4c](https://github.com/biocypher/biochatter/pull/157/commits/c137e4c5508f4d25e687d70d200cc78510e6e283)